packages/react-component-library/src/components/DatePicker/utils/isDateDisabled.test.ts

```diff
@@ -0,0 +1,73 @@
+import { isDateDisabled } from './isDateDisabled'
+
+describe('isDateDisabled', () => {
+  it('Function should return true when a date is disabled', () => {
+    const date = new Date(2024, 6, 19)
+    const disabledDays = [new Date(2024, 6, 19)]
+
+    const result = isDateDisabled(date, disabledDays)
+
+    expect(result).toBe(true)
+  })
+})
+
+it('Function should return false when a date is not disabled', () => {
```

Comment:
Indentation is wrong here - do you have prettier installed and setup to apply on save?

Reply:
I do, they just weren't nested inside of the describe block